README.md

```diff
@@ -110,6 +110,12 @@ Here are the minimal requirements to use fast-check properly without any polyfil
 
 (1) Except for features that cannot be polyfilled - such as `bigint`-related ones - all the capabilities of fast-check should be usable given you use at least the minimal recommended version of node associated to your major of fast-check.
 
+## ReScript bindings

```

Comment:
I'd rather put it directly into the Compatibility section with a bold label, something like
```suggestion
**ReScript bindings**

```

Reply:
Fair enough!